WzComparerR2.MapRender/FrmMapRender2.SceneRendering.cs

```diff
@@ -195,6 +195,18 @@ private void OnSceneItemClick(SceneItem item)
                 {
                     MoveToPortal(portal.ToMap, portal.ToName, portal.PName);
                 }
+                else if (portal.GraphTargetMap.Count == 1)
+                {
+                    MoveToPortal(portal.GraphTargetMap[0], "sp", portal.PName);
+                }
+                else if (portal.GraphTargetMap.Count > 1)
+                {
+                    this.ui.Teleport.Sl = this.StringLinker;
+                    this.ui.Teleport.CmbMaps.ItemsSource = null;
+                    this.ui.Teleport.CmbMaps.ItemsSource = portal.GraphTargetMap;
```

Comment:
![2024_2_9_4](https://github.com/Kagamia/WzComparerR2/assets/101188238/bf0f8e03-ee10-4d52-9ac5-4d82a7fb1a5b)
Because of minor unintended focus..?
Idk how to fix it

Reply:
`this.ui.Teleport.CmbMaps.ItemsSource = portal.GraphTargetMap.ToList();` 👍

Also moving `this.ui.Teleport.Sl = this.StringLinker;` to Initialize() of FrmMapRender2.cs seems better.